resources/views/livewire/auth/login.blade.php

```diff
@@ -29,20 +30,55 @@ public function login(): void
 
         $this->ensureIsNotRateLimited();
 
-        if (! Auth::attempt(['email' => $this->email, 'password' => $this->password], $this->remember)) {
+        // Get the user model from auth config in a single line
+        $userAttemptingLogin = app(config('auth.providers.users.model'))::where('email', $this->email)->first();
+
+        if (! $userAttemptingLogin || ! \Illuminate\Support\Facades\Hash::check($this->password, $userAttemptingLogin->password)) {
```

Comment:
This skips the Timebox for timing attack enumeration protection, making it trivial to identify is a user account exists or not in the application.

Reply:
Can you elaborate on this more. Is there something that needs to be updated for security reasons? Here, we want to check the user's password, rather than running `Auth::attempt`, because `Auth::attempt` will automatically log the user in, and we don't want to do that yet if two-factor authentication is enabled.

Let me know. Thanks 😊